basics.py

```diff
@@ -0,0 +1,115 @@
+import pandas as pd 
+import matplotlib.pyplot as plt
+import numpy as np
+from pandas_datareader import data as web
+from scipy import stats
+import seaborn as sns
+from datetime import datetime
+
+pd.options.display.float_format = '{:.5f}'.format
+
+def get_data_close(symbol, start, end):
+    df = pd.DataFrame(web.DataReader(symbol, data_source='yahoo', start = start, end = end)['Adj Close'])
+    return df
+
+def changeCol(df, col, name):
+    df = df.rename(columns = {col : name})
+    return df
+
+def norm_dist(symbol, start,end =  datetime.today().strftime('%Y-%m-%d'),plot = False):
```

Comment:
The time format is repeatedly used through functions. Perhaps we could make this into a constant?

Reply:
Yeah we can make it a constant. It essentially uses today's date as default end date if the user does not define any end date to the date-range. We can definitely make a constant.